serendipity_event_markdown/serendipity_event_markdown.php

```diff
@@ -237,6 +251,49 @@ function _markdown_markup($text) {
         return Markdown($text);
     }
     */
+
+    function addToCSS(&$eventData) {
+        $eventData .= '
+/* Footnotes (generated by serendipity_event_markdown) */
+
+a.footnote-ref:after {
+  content: ")";
+}
+
+.footnotes hr {
+  border-top: dashed #cccccc;
```

Comment:
I prefer the long form as I find it easier to read, but will adhere to your suggestion. :)